BrandonBeltran/hackerrank/03/JavaList.java

```diff
@@ -0,0 +1,47 @@
+package bootcamp;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Scanner;
+
+public class JavaList {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int size = scanner.nextInt();
+        List <Integer> numbers = new ArrayList<>();
```

Comment:
Best practice in Collections or Map Variable Name, is to use singular name and them interface name: numberList, change it in all variables